pkg/cmd/server/origin/handlers.go

```diff
@@ -155,10 +154,12 @@ func (c *MasterConfig) versionSkewFilter(handler http.Handler, contextMapper api
 	})
 }
 
+const legacyImpersonateUserScopeHeader = "Impersonate-User-Scope"
```

Comment:
Godoc explaining that older servers respected this.  We switched to the new way in 1q 2017.

Reply:
ok